torch/fx/experimental/unification/multipledispatch/core.py

```diff
@@ -77,7 +77,7 @@ def ismethod(func):
         return signature.parameters.get('self', None) is not None
     else:
         if sys.version_info.major < 3:
```

Comment:
Isn't this code just dead since we don't support Python 2? Just remove it.

Reply:
I'm trying hard to not make actual code changes to keep the PRs less risky haha